liqwid-plutarch-extra.cabal

```diff
@@ -160,10 +160,16 @@ library
 
 -- Test suite
 
-test-suite properties
+test-suite test
   import:         common-lang
   type:           exitcode-stdio-1.0
   main-is:        Main.hs
+  other-modules:
+    Properties.FunctorProp
+    Properties.NumericProp
+    Properties.OrdProp
+    Properties.TraversableProp
```

Comment:
The `Prop` suffix is redundant. I'd even go so far as not bothering to have more than a single-layer hierarchy: we're not exporting this publically, no chance of clashing with anything.